client/lib/purchases/index.js

```diff
@@ -213,7 +213,11 @@ function showCreditCardExpiringWarning( purchase ) {
 }
 
 function showEditPaymentDetails( purchase ) {
-	return ! isExpired( purchase ) && ! isOneTimePurchase( purchase ) && ! isIncludedWithPlan( purchase ) && isPaidWithCreditCard( purchase );
+	return showEditPaymentDetailsOption( purchase ) && isPaidWithCreditCard( purchase );
+}
+
+function showEditPaymentDetailsOption( purchase ) {
```

Comment:
I'm having trouble understanding why it is necessary to have both of these functions. Could you explain it?

Also, what do you think about moving these functions to `ManagePurchase`? Rationale:

- They relate directly to presentation logic.
- They're only used in that component.

It also might be nice to rename them, because they don't show anything. `canEditPaymentDetails`?